lib/css_parser/rule_set.rb

```diff
@@ -42,12 +66,11 @@ def get_value(property)
       return '' unless property and not property.empty?
 
       property = property.downcase.strip
-      properties = @declarations.each_with_object(String.new) do |(key, data), val|
+      properties = @declarations.each_with_object([]) do |(key, data), val|
         # puts "COMPARING #{key} #{key.inspect} against #{property} #{property.inspect}"
-        importance = data[:is_important] ? ' !important' : ''
-        val << "#{data[:value]}#{importance}; " if key.downcase.strip == property
+        val << data if key.downcase.strip == property
       end
-      properties ? properties.strip : ''
+      properties.any? ? "#{properties.map(&:to_s).join('; ')};" : ''
```

Comment:
any checks each element, so faster to use empty
```suggestion
      properties.empty? ? '' : "#{properties.map(&:to_s).join('; ')};"
```